lib/parsers/font_weight_parser.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:design_tokens_builder/parsers/design_token_parser.dart';
+
+/// Parses tokens of type `fontWeights` to Flutter code.
+///
+/// The font weight needs to be part of the specified [_allowedWeights] in order
+/// to be parsed correctly.
+///
+/// E.g.
+/// Figma design tokens:
+///   "value": "400"
+///
+/// Flutter generated code:
+///   FontWeight.w400
+class FontWeightParser extends DesignTokenParser {
+  /// Constructs a [FontWeightParser].
+  FontWeightParser([super.indentationLevel, super.config]);
+
+  @override
+  List<String> get tokenType => ['fontWeights'];
+
+  @override
+  String get flutterType => 'FontWeight';
+
+  final _allowedWeights = [100, 200, 300, 400, 500, 600, 700, 800, 900];
+
+  @override
+  String buildValue(value) {
+    if (value is String) {
+      final abs = double.parse(value).toInt();
```

Comment:
Can it ever really be double ?

```suggestion
      final abs = int.parse(value);
```

Reply:
Probably not :D

Reply:
Just checked it. Actually we do need this since in preperation we convert any "int" to a double due to math expressions 😅 So would be needed here.